src/System.Management.Automation/engine/hostifaces/Command.cs

```diff
@@ -497,6 +509,16 @@ CommandOrigin origin
                     commandProcessorBase = new CommandProcessor(functionInfo, executionContext,
                                                                 _useLocalScope ?? false, fromScriptFile: false, sessionState: executionContext.EngineSessionState);
                 }
+                else if (ScriptPath != null)
+                {
+                    var scriptName = Path.GetFileName(ScriptPath);
+                    var scriptInfo = new ExternalScriptInfo( scriptName, ScriptPath, executionContext);
+                    commandProcessorBase = new DlrScriptCommandProcessor( scriptInfo, 
+                        executionContext,_useLocalScope ?? false,
+                        executionContext.EngineSessionState);
```

Comment:
Formatting:
```suggestion
                    commandProcessorBase = new DlrScriptCommandProcessor(
                        scriptInfo, 
                        executionContext,
                        _useLocalScope ?? false,
                        executionContext.EngineSessionState);
```

Reply:
This is the same kind of case like above - I did it first the way you suggest but then I reformatted it to keep the convention already used - [see this call](https://github.com/PowerShell/PowerShell/blob/16ec1cc32f9e60c9d76b074c865c32a5d2acf5a2/src/System.Management.Automation/engine/hostifaces/Command.cs#L502-L506) a few lines below. 

So the general approach is to ignore conventions already used and use formatting as suggested by CodeFactor analyzer? In that case should I also change the comment for the ScriptPath property to start with word 'Get' even though comments for other properties in that class start with word 'Access'?

Reply:
@m5x yeah there's some older code in a fair few places that we're more or less trying to update as we come across it. In general prefer the newer syntax.

As for the doc comments, I'd think we should go with the CodeFactor recommendation there. Since they're only comments, too, is it worth updating the other non-compliant documentation comments in this file do you think @iSazonov?

Reply:
It is not strong request - we could fix formatting later. But the PR is small and the code was already changed so we could fix formatting now.

Reply:
Fixed with last commit. Also fixed the property docs in this class which did start with word 'access' instead of 'gets'.